seqr/migrations/0070_remove_rnasample_dataset_type_and_more.py

```diff
@@ -0,0 +1,123 @@
+# Generated by Django 4.2.13 on 2024-07-12 20:14
+
```

Comment:
this migration is split into 2 separate migrations because due to limitations in Django it is not possible to split the data into 2 tables and then rename those tables in the same migration